packages/wrangler/src/d1/info.tsx

```diff
@@ -45,16 +47,88 @@ export const Handler = withConfig<HandlerOptions>(
 				},
 			}
 		);
+
+		let output: Record<string, string | number> = { ...result };
+		if (output["file_size"]) {
+			output["database_size"] = output["file_size"];
+			delete output["file_size"];
+		}
+		if (result.version === "beta") {
+			const today = new Date();
+			const yesterday = subDays(today, 1);
```

Comment:
This is overkill to install and use the date-fns library just for this `subDays()` function. In this context it is equivalent to:

```
yesterday = new Date();
yesterday.setDate(yesterday.getDate() - 1);
```

See https://github.com/date-fns/date-fns/blob/e36a05cea15fda926ea93108f661e0aed4a4bed9/src/addDays/index.ts#L31

Reply:
It's more like:
```
const today = new Date();
const yesterday = new Date(new Date(today).setDate(today.getDate() - 1));
```

But sure, can do